test/utils.py

```diff
@@ -290,3 +290,54 @@ def pfb_job_status_in_terra(workspace, job_id):
     else:
         print(resp.content)
         resp.raise_for_status()
+
+
+def add_requester_pays_arg_to_url(url, billing_project='drs-billing-project'):
+    endpoint, args = url.split('?', 1)
+    return f'{endpoint}?userProject={billing_project}&{args}'
+
+
+@retry(error_codes={500, 502, 503, 504})
+def import_drs_from_gen3(drs: str) -> requests.Response:
+    """
+    Import the first byte of a DRS URI using gen3.
+
+    Makes two calls, first one to gen3, which returns the link needed to make the second
+    call to the google API and fetch directly from the google bucket.
+    """
+    if drs.startswith('drs://'):
+        drs = drs[len('drs://'):]
```

Comment:
Gen3 refers to objects like `dg.4503/4c161631-ee81-486c-94ad-85b83272eafd` as GUIDs

Reply:
Good point.  I think I was slightly confused.  I'll update with the term GUID.